AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -243,13 +248,18 @@ - (void)createWithPartition:(NSString *)partition
                                                                  }
                                                                  MSLogDebug([MSDataStore logTag], @"Document json:%@", json);
 
+                                                                 // Create an instance of Document.
+                                                                 // TODO fix the warning.
+                                                                 Class aClass = [(NSObject *)document class];
```

Comment:
You don't need to cast to `(NSObject *)`. Instead make the MSSerializableDocument protocol inherit from the NSObject protocol